core/src/main/java/nl/nn/adapterframework/ldap/LdapQueryPipeBase.java

```diff
@@ -102,7 +142,15 @@ protected String retrieveUrl(String host, int port, String baseDN, boolean useSs
 		return url + d;
 	}
 
-
+	@Override
+	public void setCache(ICacheAdapter<String, Set<String>> cache) {
+		this.cache=cache;
+	}
+	@Override
+	public ICacheAdapter<String, Set<String>> getCache() {
+		return cache;
+	}
```

Comment:
Dit soort simpele getters en setters graag via Lombok annotaties '@Getter' en '@Setter'

Reply:
Voor alle in deze PR toegevoegde/verplaatste attributen die zulke simpele getters en setters heeft toegevoegd.